youtube_dl/extractor/soundcloud.py

```diff
@@ -350,6 +350,8 @@ def add_format(f, protocol, is_preview=False):
             format_id_list = []
             if protocol:
                 format_id_list.append(protocol)
+            if f.get('ext') == 'aac':
+                f['abr'] = '256'
```

Comment:
Must be numeric not string.

Reply:
@dstftw String is converted to int on line 364. At line 354 all other extensions' bit rates are as strings, that's how they're passed into `def add_format()`. Also by **account-needed** do you mean that Auth Token is not acceptable and that username and password also need supporting?

Reply:
Meaning that you must provide account credentials for testing or whatever else is needed.

Reply:
I can confirm that this code works by providing the OAuth token.

eg:

``` sh
 youtube-dl -f bestaudio --add-header "Authorization:OAuth <OAUTH TOKEN>" https://soundcloud.com/turborecordings/azari-iii-hungry-for-the
-5
youtube-dl -f http_aac_256 --add-header "Authorization:OAuth <OAUTH TOKEN>" https://soundcloud.com/turborecordings/azari-iii-hungry-for-
the-5
```

You can get this code by opening up dev tools in the browser.